### PR TITLE
Fix warning issue on recent sklearn version

### PR DESCRIPTION
**Pull request by djKooks**
Hello.
This is update to remove warnings I found while testing example. It is related with issue [here](https://github.com/scikit-learn/scikit-learn/issues/12370).

Please look on. Thanks.

### CODE REVIEW

File: examples/sklearn_additional_args.py [hunk 1]
```diff
@@ -32,7 +32,8 @@ def __call__(self, trial):
             rf_max_depth = int(trial.suggest_loguniform('rf_max_depth', 2, 32))
             classifier_obj = sklearn.ensemble.RandomForestClassifier(max_depth=rf_max_depth)
 
-        score = sklearn.model_selection.cross_val_score(classifier_obj, x, y, n_jobs=-1)
+        cv = sklearn.model_selection.ShuffleSplit()
+        score = sklearn.model_selection.cross_val_score(classifier_obj, x, y, n_jobs=-1, cv=cv)
```
Comment (sile) [inline examples/sklearn_additional_args.py]: By executing the original example (`master` branch), I got warning messages as the following:
```console
$ python -c 'import sklearn; print(sklearn.__version__)'
0.20.2

$ python optuna/examples/sklearn_simple.py
/...abbrev.../_split.py:2053: FutureWarning: You should specify a value for 'cv' instead of relying on the default value.  \
The default value will change from 3 to 5 in version 0.22.

...abbrev...
```

The message and [scikit-learn's documentation(v0.20)](https://scikit-learn.org/0.20/modules/generated/sklearn.model_selection.cross_val_score.html) say "the default value of `cv` parameter is `3`". 
Although your code seems to work, I think it is better to specify `cv=3` instead of `cv=ShuffleSplit()`, because it is consistent with the behavior of the original example.

File: examples/sklearn_simple.py [hunk 2]
```diff
@@ -38,7 +38,8 @@ def objective(trial):
         rf_max_depth = int(trial.suggest_loguniform('rf_max_depth', 2, 32))
         classifier_obj = sklearn.ensemble.RandomForestClassifier(max_depth=rf_max_depth)
 
-    score = sklearn.model_selection.cross_val_score(classifier_obj, x, y, n_jobs=-1)
+    cv = sklearn.model_selection.ShuffleSplit()
+    score = sklearn.model_selection.cross_val_score(classifier_obj, x, y, n_jobs=-1, cv=cv)
```
Comment (sile) [inline examples/sklearn_simple.py]: ditto.

File: examples/sklearn_additional_args.py (hunk 1 above)
Comment (djKooks) [reply examples/sklearn_additional_args.py]: @sile thanks! Just updated.

**Comment by djKooks** [comment]
@sile thanks for guide! I just updated it, so please look on.
